Makefile

```diff
@@ -120,6 +122,16 @@ clean:
 	rm -rf $(DIST) pkg/pillar/Dockerfile pkg/qrexec-lib/Dockerfile pkg/qrexec-dom0/Dockerfile \
 	       images/installer.yml images/rootfs.yml
 
+yetus-image:
+	docker build -t $(YETUS_IMG) ./yetus
+
+yetus: yetus-image
```

Comment:
Please make sure to add yetus target to the help section of the Makefile.

And speaking of help section -- I'm not clear at all about what this target is expected to do. The way we use yetus today is for *patch* (pull request) testing. It is not clear to me at all -- what is the notion of a *patch* when you call this target and what base it has to be applied to.

Reply:
> Please make sure to add yetus target to the help section of the Makefile.

Done

> what is the notion of a patch when you call this target and what base it has to be applied to

Ah, now that is a good question. 

The goal here is to be able to run it locally (secondary benefit is simplifying circleci config so it is a simple `run`, with everything else portable, but very much secondary).

I hadn't considered that it applies to patches themselves. I would have expected it to be like a lint or similar, which just applies to _everything_, not just changes. I never actually realized that it is a diff checker. I am guessing we can apply it to the diff from the base of the current branch, but let's resume this conversation below with AW.